Services/LookupService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using Domain;
+using PimsExporter.Services;
+using PimsExporter.Services.InputRepositories;
+
+namespace Services
+{
+    public class LookupService : ILookupService
+    {
+        public Dictionary<string, string> GetSapIds(IRootSiteRepository rootSiteRepository)
```

Comment:
This method should move into RootRepository. Use linq instead of foreach

Reply:
Done